app/web_framework/cluster_master_application.py

```diff
@@ -152,6 +156,51 @@ def get(self, build_id, subjob_id, atom_id):
         self.write(response)
 
 
+class _AtomConsoleHandler(_ClusterMasterBaseAPIHandler):
```

Comment:
@josephharrington - I wasn't sure whether this was appropriate or not to put inside the ClusterMaster class, vs outside.

The whole 'except' clause in the try/except contains a redirect, which I wasn't sure if we wanted to keep out of the ClusterMaster class and contained within the web framework code (ClusterMasterApplication).